korman/exporter/material.py

```diff
@@ -147,6 +147,16 @@ def __init__(self, exporter):
             "transformCtl": self._export_layer_transform_animation,
         }
 
+    def calc_material_ambient(self, bo, bm) -> hsColorRGBA:
+        emit_scale = bm.emit * 0.5
```

Comment:
OK, not sure how this is working (the color is odd in PRPShop), but this appears to correct the colors on my end. Maybe give it a whirl and see how it does?:

```
        emit_scale = (bm.emit / 10)
        if emit_scale > 0.0:
            return hsColorRGBA(bm.diffuse_color.r * (emit_scale + 1.0) + 0.1,
                               bm.diffuse_color.g * (emit_scale + 1.0) + 0.1,
                               bm.diffuse_color.b * (emit_scale + 1.0) + 0.1,
                               1.0)
```

Reply:
Per Hoikas' suggestion on IRC of tying shadeless into this equation, I tried this:

```
        if bm.emit > 0.0 or bm.use_shadeless == True:
            return hsColorRGBA(bm.diffuse_color.r + 0.175,
                               bm.diffuse_color.g + 0.175,
                               bm.diffuse_color.b + 0.175,
                               1.0)
```
Seems to work about the same if we're using Cyan's Ages.

Reply:
That seems like it could cause issues for brighter colours, potentially pushing them above 1.0?

I'm confused why we can't just use the diffuse colour?

Reply:
I was hoping that avoiding the multiplication (and comparison) against `emit` would fix the colors. We still may have to assign the diffuse value to either `runtime` or `preshade`... Even though PlasmaMax does not...

Reply:
Using the diffuse color straight up causes the flares to be a slightly darker shade from their original counterparts...in this case, Kirel. It's really noticeable with the blue light garden lamp flares. The math above got them a lot closer FWIW.